### PR TITLE
Nopager

### PR DESCRIPTION
**Pull request by fraoustin**
## Description

manage command \pset pager [on/off] 
set \pset pager off next output don't use pager
set \pset pager on next output use pager

Add parameter in section "main" : pset-pager.
When I want to start pgcli with pset-pager is off,  output don't use pager

I's is a partial response for issue 813

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: pgcli/pgclirc [hunk 1]
```diff
@@ -131,6 +131,9 @@ multiline_continuation_char = ''
 # The string used in place of a null value.
 null_string = '<null>'
 
+# manage pager on startup
+pset-pager = on
```
Comment (amjith) [inline pgcli/pgclirc]: Let's change the name to `enable_pager` and use the values `True` and `False`. 

Let's move this option close to the `pager` option in the config file (line 76).

**Comment by j-bennet** [comment]
@fraoustin Perhaps be a little more descriptive with commit comments?

I.e., "enable_pager now accepts on/off" is better than "Update main.py".

:)

**Comment by fraoustin** [comment]
It's ok for you?

**Comment by j-bennet** [comment]
Looks good, I'll try this out.

**Comment by j-bennet** [comment]
Woks fine. Merging!

Thank you for the PR!

**Pull request merged**